docs/build/quickstarts/connecting_to_the_world.md

```diff
@@ -53,319 +41,178 @@ This clearly is a much more minimalist scaffold than what the *quickstart* templ
-}
- elapsed time: 1.868767541s
-
-2>
-```
-
-Again, our HTTP request successfully executed but now we can read our *stderr* and *stdout* pipes a little better. Loos like the curl binary writes the request metadata into *stderr* and we get the html code of the *example.com* site in *stdout*. Of course, go ahead, double check the output in your browser.
-
-Before we get to implement our actual business logic utilizing the curl-adapter module, lets quickly recap what happened so far: We created a reusable(!) Wasm module that allows us to bind to a host's curl binary and learned that Marine enables the necessary FFI approach to bind to essentially any binary available and permissioned by a host including the curl binary. That by the way is actually a powerful construct adding much value to the Wasm runtime and, of course, your Fluence Functions. Finally, we tested our curl binding to the local host and experimented with both byte- and string-based return types.
-
-Let's kick it up a notch and turn our sleepy greeting function into something a little more interesting and useful. How about we create a Fluence Functions that takes a timezone instead of just a name and returns the local date and time? To make things a little easier on us, we'll use the [WorldTimeAPI](https://worldtimeapi.org/) to do the heavy lifting and provide us with the desired information for a given timezone parameter.
-
-Change into the `src/services/http_requester/modules/http_requester` to add a couple Rust crates we want to use in our new endeavor with `cargo add chrono serde_json` and proceed to our code, which goes into the replaces the template code in `<..>/modules/http_requester/src/main.rs`:
-
-```rust
-#![allow(non_snake_case)]
-use marine_rs_sdk::{marine, MountedBinaryStringResult};
-use serde_json;
-
-// url for the Worldtime API
-const API_URL: &str = "https://worldtimeapi.org/api/timezone";
-
-
-pub fn main() {}
-
-// custom struct to return our query results in form of stderr, stdout
-// making it easier to subsequently handle the output
-#[marine]
-pub struct SimpleResult {
-    pub stderr: String,
-    pub stdout: String,
-}
+All looks good and we're ready to deploy our code. Note that we had to add the `{"particle": {"id":"id", "token":"token"}}` configuration map to our call to handle the vault references expected by the module as there is no particle available in REPL mode.
 
-impl SimpleResult {
-    fn new( stderr: String, stdout: String) -> Self {
-        SimpleResult {
-            stderr,
-            stdout,
-        }
-    } 
-}
+As a last step, deploy your service to either `dar` or `local` and let's update our `aqua.main` so we can run the deployed code:
 
-// mostly for demo purposes, note how this fucntion is not wrapped in marine
-// and therefore not available to be called. check the REPL output.
-fn get_timezone(timezone: &String) -> SimpleResult {
+```python
```

Comment:
why python? :)
```suggestion
```aqua
```

Reply:
much nicer hghlights

Reply:
Nicer then our official highlights that are used in vscode and everywhere else in the docs?

Reply:
Ah you probably mean the syntax highlighting in your editor - this will be without syntax highlighting, yeah. But I made it so in docusaurus our vscode syntax highlighting is used, which looks great in my opinion, if you agree - let's do dedicated aqua code blocks in docusaurus

Reply:
yes. sounds good.